numpy/core/tests/test_unicode.py

```diff
@@ -4,40 +4,28 @@
 from numpy.compat import unicode
 from numpy.testing import assert_, assert_equal, assert_array_equal
 
-# Guess the UCS length for this python interpreter
-if sys.version_info[:2] >= (3, 3):
-    # Python 3.3 uses a flexible string representation
-    ucs4 = False
-
-    def buffer_length(arr):
-        if isinstance(arr, unicode):
-            arr = str(arr)
-            if not arr:
-                charmax = 0
-            else:
-                charmax = max([ord(c) for c in arr])
-            if charmax < 256:
-                size = 1
-            elif charmax < 65536:
-                size = 2
-            else:
-                size = 4
-            return size * len(arr)
-        v = memoryview(arr)
-        if v.shape is None:
-            return len(v) * v.itemsize
+# Python 3.3+ uses a flexible string representation
+ucs4 = False
```

Comment:
I have a note to simplify this and a couple of other things later

Reply:
Perhaps worth tagging all of this type of thing with a `# TODO[gh-####]` in the source, where `gh-####` is some tracking issue that may or may not already exist.

That way, if you forget later, someone else will be able to work it out.